IaC/BareMetal/README.md

```diff
@@ -38,116 +38,33 @@ az monitor log-analytics workspace create -g $Ngsa_Log_RG -n $Ngsa_Log_Name -l $
 
 ```
 
-## Create VM
+## Create Azure VM
 
-Create your VM per instructions in [Bare Metal Setup](setup-bare-metal-vm.md)
```

Comment:
I modified this to point to the stand-alone repo for setting up Azure Kubernetes dev cluster. We should eventually move this to github/microsoft